go/pkg/tool/tool_test.go

```diff
@@ -109,55 +112,98 @@ func TestTool_CheckDeterminism(t *testing.T) {
 	e, cleanup := fakes.NewTestEnv(t)
 	defer cleanup()
 	cmd := &command.Command{
-		Args:        []string{"foo bar baz"},
+		Args:        []string{"foo", "bar", "baz"},
```

Comment:
This is here (and below) in order to not provide bad examples in our unit tests. E.g. "echo hello world" would not work as a single string; it needs to be split up.